packages/controls/src/widget_button.ts

```diff
@@ -101,10 +101,15 @@ export class ButtonView extends DOMWidgetView {
   update(): void {
     this.el.disabled = this.model.get('disabled');
     this.updateTabindex();
-    this.el.setAttribute('title', this.model.get('tooltip'));
 
+    const tooltip = this.model.get('tooltip');
     const description = this.model.get('description');
     const icon = this.model.get('icon');
+
+    tooltip
+      ? this.el.setAttribute('title', tooltip)
+      : this.el.setAttribute('title', description);
```

Comment:
We can express very similar logic using the nice ?? operator in one line. One advantage to using ?? is that you can do `b.tooltip = ''` to turn off the tooltip if you want (i.e., setting tooltip to None will default to the description, but if tooltip is a string we use it.)

```suggestion
    this.el.setAttribute('title', tooltip ?? description);
```

Reply:
Ah very nifty. I don't see any reason to change your suggestion. Seems like a good opportunity to use the "Commit Suggestion" for my first time :)